appyters/L1000KD2/fields/AutocompleteField.py

```diff
@@ -0,0 +1,25 @@
+from appyter.fields import Field 
+import json
+import requests
+
+class AutocompleteField(Field):
+    """
+    Representing field that allows search for thousands of possible RNA genes.
+    :param name: (str) A name that will be used to refer to the object as a variable and in the HTML form.
+    :param label: (str) A human readable label for the field for the HTML form
+    :description: (Optional[str]) A long human readable description for the field for the HTML form
+    :param default: (float) A default value as an example and for use during prototyping
+    :param section: (Optional[str]) The name of a SectionField for which to nest this field under, defaults to a root SectionField
+    :param value: (INTERNAL Any) The raw value of the field (from the form for instance)
+    """
+    def __init__(self, **kwargs):
+        kwargs["choices"] = None 
+        super().__init__(**kwargs)  
+    
+    def constraint(self):
+        # file_path = self.args['file_path']
+        # data = json.loads(requests.get(file_path).text)
+        # key = list(data.keys())[0]
+        # values = data[key]
+        # return self.raw_value is not None and self.raw_value in values
+        return True
```

Comment:
Please add a constraint to your field for security reasons. If you don't want to base it off of the actual file_path, at least provide a string based constraint like only alpha-numerics.

Reply:
I was also experimenting with the autocomplete field. I found that when the constraint originally written is included, the appyter form doesn't submit properly. For me, the autocomplete field appears to work properly, but when I try to submit the form, nothing happens. I haven't figured out a solution yet.

Reply:
Firstly, you can use `requests.get(file_path).json()` as a shorthand. But running it myself in terminal, I don't see an issue: 
`requests.get('https://appyters.maayanlab.cloud/storage/L1000FWD/CREEDS_info.json').json().keys()`. Is it possible you do not have the `requests` package installed?